f.services.md

```diff
@@ -50,6 +50,16 @@ kubectl run busybox --rm --image=busybox -it --restart=Never --env="IP=$IP" -- w
 # Tip: --timeout is optional, but it helps to get answer more quickly when connection fails (in seconds vs minutes)
 ```
 
+</p>
```

Comment:
the question asks for Cluster IP though - maybe we can make this a separate question, if it doesn't exist already?